python/cudf/cudf/core/dataframe.py

```diff
@@ -3748,16 +3748,23 @@ def merge(
             If on is None and not merging on indexes then
             this defaults to the intersection of the columns
             in both DataFrames.
-        how : {‘left’, ‘outer’, ‘inner’}, default ‘inner’
+        how : {‘left’, ‘outer’, ‘inner’, 'leftsemi', 'leftanti'}, \
+            default ‘inner’
             Type of merge to be performed.
 
             - left : use only keys from left frame, similar to a SQL left
               outer join.
             - right : not supported.
             - outer : use union of keys from both frames, similar to a SQL
               full outer join.
-            - inner: use intersection of keys from both frames, similar to
+            - inner : use intersection of keys from both frames, similar to
               a SQL inner join.
+            - leftsemi : similar to ``inner`` join, but in addition returns all
+              columns from the left dataframe and ignores all columns from
+              the right dataframe.
```

Comment:
I think the "in addition" phrasing is a bit confusing.
```suggestion
            - leftsemi : similar to ``inner`` join, but in addition returns all
              columns from the left dataframe and ignores all columns from
              the right dataframe.
```
```suggestion
            - leftsemi : similar to ``inner`` join, but only returns columns
               from the left dataframe and ignores all columns from the
               right dataframe.
```

Reply:
Sounds good to me, thanks @vyasr